docs/usage/self-hosted-configuration.md

```diff
@@ -7,6 +7,41 @@ description: Self-Hosted Configuration usable in renovate.json or package.json
 
 The below configuration options are applicable only if you are running your own instance ("bot") of Renovate.
 
+## allowPostUpgradeCommandTemplating
+
+If true allow templating for post-upgrade commands.
+
+Let's look at an example of configuring packages with existing Angular migrations.
+
+Add two properties to `config.js`: `allowPostUpgradeCommandTemplating` and `allowedPostUpgradeCommands`
+
+```javascript
+module.export = {
+  allowPostUpgradeCommandTemplating: true,
+  allowedPostUpgradeCommands: ['^npm ci --ignore-scripts$', '^npx ng update'],
+};
+```
+
+In the `renovate.json` file, define the commands and files to be included in the final commit.
+
+```json
+{
+  "postUpgradeTasks": {
+    "commands": [
+      "npm ci --ignore-scripts",
+      "npx ng update {{{depName}}} --from={{{fromVersion}}} --to={{{toVersion}}} --migrateOnly --allowDirty --force"
+    ],
+    "fileFilters": ["**/**"]
+  }
+}
+```
+
+With this configuration, the executable command for `@angular/core` will look like this
```

Comment:
This config will run for all packages, but it doesn't harm, angular cli will handle it.

Reply:
changed a configuration only for angular/core